pkg/util/util.go

```diff
@@ -139,7 +139,7 @@ func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
 // accept a name or number.
 type IntOrString struct {
 	Kind   IntstrKind
-	IntVal int
+	IntVal int32
```

Comment:
It is really weird to see code like int(blah.IntVal). I can live with it if I have to, but I'd prefer not to...

* Leave as int, return error if it's > 2B when marshalling? (UGH)
* Change name to Int32Val? 
* Add a `Int() int` function that does the cast for you?